client/core/core.go

```diff
@@ -2317,6 +2317,10 @@ func (c *Core) loadWallet(dbWallet *db.Wallet) (*xcWallet, error) {
 		})
 	}
 	if err != nil {
+		if errors.Is(err, asset.ErrWalletTypeDisabled) {
+			subject, details := c.formatDetails(TopicWalletTypeDeprecated, unbip(assetID))
+			c.notify(newWalletConfigNote(TopicWalletTypeDeprecated, subject, details, db.WarningLevel, nil))
+		}
 		return nil, fmt.Errorf("error opening wallet: %w", err)
 	}
```

Comment:
I started into a scheme where we just disable the wallet, but it was a can of worms. Instead, we're refusing to open a deprecated wallet in e.g. `eth.NewWallet` and issuing a warning. This is no problem for us right now, because the only other option for Ethereum is also a seeded wallet and will derive the same key. But if there was a need to export a seed from a deprecated wallet, we would need to get more sophisticated here.